brian2/importexport/importexport.py

```diff
@@ -0,0 +1,74 @@
+import abc
+from abc import abstractmethod, abstractproperty
+
+class ImportExport(object):
+    __metaclass__ = abc.ABCMeta
+    
+    #: A dictionary mapping import/export methods names to `ImportExport` objects
+    methods = dict()
+
+    @staticmethod    
+    def register(name, importerexporter):
+        '''
+        Register a import/export method. Registered importerexporter can be referred to
+        via their name.
+        
+        Parameters
+        ----------
+        name : str
+            A short name for the import/export class (e.g. `'dict'`)
```

Comment:
Need double-backtick quotes here in RST for `'dict'` (single backticks are to refer to existing classes/functions)